instancio-core/src/main/java/org/instancio/generator/specs/bra/CpfGeneratorSpec.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.instancio.generator.specs.bra;
+
+import org.instancio.generator.specs.NullableGeneratorSpec;
+
+/**
+ * Spec for generating <a href="https://en.wikipedia.org/wiki/CPF_number">
+ * BRA 'Cadastro de Pessoas Físicas' </a> (CPF).
+ *
+ * @since 5.0.0
+ */
+public interface CpfGeneratorSpec extends NullableGeneratorSpec<String> {
+
+    /**
+     * Format the CPF output.
+     * For example, if the format is {@code true}, the result will be formatted as follows: {@code '123.435.983-09'}
+     * In case the format is {@code false}, the result will be formatted as follows: {@code '12343598309'}
+     *
+     * @param format for separating groups of digits,
+     *                  default is {@code false} (groups are not separated)
+     * @return spec builder
+     * @since 5.0.0
+     */
+    CpfGeneratorSpec format(Boolean format);
```

Comment:
I think we can avoid the boolean parameter here and instead have the following method signatured:

```java
    CpfGeneratorSpec formatted();
```

In other words, by default, the value is not formatted, and by using the `formatted()` method we can generate formatted values.

Reply:
Changed!